.github/workflows/linux_build_and_test.yml

```diff
@@ -0,0 +1,45 @@
+name: Linux Build and Test
+
+on:
+  workflow_call:
+    inputs:
+      amdgpu_families:
+        type: string
+      artifact_run_id:
+        type: string
+      test_runs_on:
+        type: string
+      run_linux_tests_only:
+        type: string
```

Comment:
Can this use `type: boolean`?

See also my other comment about name, and could drop "linux" from this, so `use_prebuilt_artifacts`